.github/workflows/goTest.yml

```diff
@@ -0,0 +1,50 @@
+name: Test
+
+on: 
+  # push:
```

Comment:
You can `push`

Reply:
The [new commit ](https://github.com/WuraLab/Boardly/pull/38/commits/88f1bfdaf3a7b8a5b15d8151e8477a45794758ad) removed it. Thanks for flagging that.